src/app/customer/getCreateCustomerValidationSchema.ts

```diff
@@ -0,0 +1,70 @@
+import { FormField, LanguageService } from '@bigcommerce/checkout-sdk';
+import { memoize } from '@bigcommerce/memoize';
+import { object, string, ObjectSchema } from 'yup';
+
+import { getCustomFormFieldsValidationSchema, CustomFormFieldValues, TranslateValidationErrorFunction } from '../formFields';
+
+import getEmailValidationSchema from './getEmailValidationSchema';
+import { PasswordRequirements } from './getPasswordRequirements';
+
+export type CreateAccountFormValues = {
+    firstName: string;
+    lastName: string;
+    email: string;
+    password: string;
+    acceptsMarketingEmails?: string[];
+} & CustomFormFieldValues;
+
+export interface CreateCustomerValidationSchema {
+    formFields: FormField[];
+    language: LanguageService;
+    passwordRequirements: PasswordRequirements;
+}
+
+function getTranslateCreateCustomerError(language?: LanguageService): TranslateValidationErrorFunction {
+    return (type, { label, min, max }) => {
+        if (!language) {
+            return;
+        }
+
+        if (type === 'required') {
+            return language.translate(`customer.required_error`, { label });
```

Comment:
I think you can just use `''` instead of back ticks because you're not interpolating anything. 🍹